packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config.vue

```diff
@@ -806,4 +804,8 @@ onUnmounted(() => {
 .use-button {
 	white-space: nowrap;
 }
+
+.popover.overlay-container > :deep(section) {
+	margin: 3rem 2.5rem 0rem 2.5rem;
+}
```

Comment:
I think this should be part of the `<tera-drilldown>` as a prop. Like a level or something.

Reply:
added